ragna/ui/__init__.py

```diff
@@ -0,0 +1 @@
+from .app import app
```

Comment:
One thing that we need to do before merge is to rename `ragna.ui` to `ragna._ui`. This should be private from the user perspective. Will do that as soon as everything else is ready.

Reply:
Done in 3a0e2f0ac118882a679fdcfa488cd92502ee276e